modules/formattable.py

```diff
@@ -10,3 +10,65 @@ def party_name(party):
       return 'Independent'
     else:
       return party
+
+def full_state_name(state_code):
+    states = {
+        'AK': 'Alaska',
+        'AL': 'Alabama',
+        'AR': 'Arkansas',
+        'AS': 'American Samoa',
+        'AZ': 'Arizona',
```

Comment:
we need to test these methods?

Reply:
Yep! We will do that when we add unit testing next.